src/actions/clusterActions.js

```diff
@@ -300,7 +300,17 @@ export function clusterLoadDetails(clusterId) {
     });
 
     var cluster;
-    var clustersApi = new GiantSwarmV4.ClustersApi();
+    var clustersApi = new GiantSwarm.ClustersApi();
+
+    clustersApi.getClusterV5(scheme + ' ' + token, 'm0ckd').then(
```

Comment:
Is this some debug / local testing output?

Reply:
yes, it shouldn't be here...